webextension/background/deviceInfo.js

```diff
@@ -1,10 +1,13 @@
-/* globals chrome */
+/* globals chrome, catcher */
 
 window.deviceInfo = (function () {
   let manifest = chrome.runtime.getManifest();
 
   let platformInfo = {};
   chrome.runtime.getPlatformInfo(function(info) {
+    if (chrome.runtime.lastError) {
```

Comment:
getPlatformInfo doesn't actually produce a promise.  It seems like some more frequently used APIs have had promises added, but many like this one haven't been updated.